web/themes/custom/westy/templates/content/node--alert--featured.html.twig

```diff
@@ -0,0 +1,38 @@
+{# Set Alert Type #}
+{% if node.field_severity.value == 20 %}
+	{% set alertType = 'warning' %}
+	{% set TypeName = 'warning' %}
+{% else %}
+	{% set alertType = 'info' %}
+	{% set TypeName = 'information' %}
+{% endif %}
+
+{% set alertIcon %}
+{% include "@westy/icon/icon.twig" with {
+    name: TypeName,
+    size: 'lg',
+  } only %}
+{% endset %}
+
+{# CLASSES #}
+{% set classes= [
+	  'bg-' ~ alertType,
+     'text-light',
+]%}
+
+{# TWIG #}
+<div class="{{ classes|join(' ')|trim }}" data-nid="{{ node.id }}" {% if node.changed %} data-changed="{{ node.changed.value }}" {% endif %}>
+	<div class="container">
+		<div class="d-flex flex-row p-3">
+			{{ alertIcon }}
+			<div>
+				<div class="fw-bold">{{ label }}</div>
+				<div class="global-alert-content">{{ content }}</div>
+				<span class="small">
+					Last updated:
```

Comment:
Not a huge deal as we won't be translating Employees—at least not to my knowledge—but remember when you have text in a twig template escape it with a translation filter.